pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
 section: docs
 ---
 
-The release notes discuss the changes impacting users and implementers:
+### Introduction
 
+Just as your schemas evolve over time, the JSON Schema specification does as well. This guide provides useful information to make upgrading your schemas across versions of the specification a breeze.
+
+Learn how to use our migration guides and tooling ([AlterSchema](https://alterschema.sourcemeta.com/)) to bring your work with you.
+
+### Keywords Overview
+
+Here is a comprehensive overview to get you going.
+
+| All Keywords            | Specification | Draft introduction | Removed | Changed                        |
+| ----------------------- | ------------- | ------------------ | ------- | ------------------------------ |
+| `$anchor`               | Core          | 2019-09            | No      | No (updated in 2020)           |
+| `$comment`              | Core          | 07                 | No     |                                |
+| `$id`                   | Core          | 06                 | No      | Replaced `id`                  |
+| `$defs`                 | Core          | 2019-09            | No      | Replaced `definitions`         |
+| `$dynamicAnchor`        | Core          | 2020-12            | No      | Replaced `$recursiveAnchor`    |
+| `$dynamicRef`           | Core          | 2020-12            | No      | Replaced `$recursiveRef`       |
+| `$recursiveAnchor`      | Core          | 2019-09            | 2020-12 | Replaced by `$dynamicAnchor`   |
+| `$recursiveRef`         | Core          | 2019-09            | 2020-12 | Replaced by `$dynamicRef`      |
+| `$ref` (legacy)         | Core          | 03                 | 2019-09 | Replaced by `$ref`             |
+| `$ref`                  | Core          | 2019-09            | No      | Replaced `$ref` (legacy)       |
+| `$schema`               | Core          | 03                 | No      | No                             |
+| `$vocabulary`           | Core          | 2019-09            | No      | No                             |
+| `additionalItems`       | Core          | 03                 | 2020-12 | Replaced by `items`            |
+| `allOf`                 | Core          | 04                 | No      | No                             |
+| `anyOf`                 | Core          | 04                 | No      | No                             |
+| `const`                 | Validation    | 06                 | No      | No                             |
+| `contains`              | Core          | 06                 | No      | No                             |
+| `contentEncoding` (legacy) | Core          | 01                 | 04      | Replaced by `media`.`binaryEncoding`                 |
+| `media`                    | Validation    | 04                 | 07      | Replaced `conentEncoding` (legacy) and replaced by `contentEncoding` and `contentMediaType` |
```

Comment:
Replaced **`conentEncoding`** (legacy) and replaced by `contentEncoding` and `contentMediaType` |

Typo - contentEncoding